library/src/main/java/com/okta/oidc/net/request/HttpRequestBuilder.java

```diff
@@ -218,6 +222,45 @@ public TokenRequest createRequest() throws AuthorizationException {
         }
     }
 
+    public static class DeviceSecretTokenExchange extends Builder<DeviceSecretTokenExchange> {
+        String mGrantType;
+        String mActorToken;
+        String mSubjectToken;
+
+        private DeviceSecretTokenExchange() {
+        }
+
+        @Override
+        DeviceSecretTokenExchange toThis() {
+            return this;
+        }
+
+        public DeviceSecretTokenExchange actorToken(String actorToken) {
+            mActorToken = actorToken;
+            return this;
+        }
+
+        public DeviceSecretTokenExchange subjectToken(String subjectToken) {
+            mSubjectToken = subjectToken;
+            return this;
+        }
+
+        @Override
+        protected void validate(boolean isConfigurationRequest) throws AuthorizationException {
+            super.validate(isConfigurationRequest);
+            if (mConfig.getAudience() == null || mConfig.getAudience().equalsIgnoreCase("")) {
```

Comment:
isEmpty() looks kind of better here (nit)

Reply:
fixed https://github.com/okta/okta-oidc-android/pull/262/commits/a5c97e4bedf174370a98369157c0f032ee193431